dataactcore/config_example.yml

```diff
@@ -33,63 +43,112 @@ broker:
     # File D1 API
     # Uses "{}" because it requires string formatting to insert query parameters
     # If the URL is provided, the file name provided will be used when uploading to S3
-    award_procurement_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}&callback={}
+    award_procurement_url: https://award_procurement_url.gov
```

Comment:
There is a large set of things we don't use anymore, I think if we're already fixing the example config, we need to yank them out. Anything to do with smartronix is probably one of those things (but give it a look before tearing it all out). Some things we no longer use have already been torn out further in the file.

It sounds like that's out of scope for this but to me that's exactly the scope. Part of the decision was to make it so you could just drag and drop the contents here to your own config, I think that involves cleaning out unneeded content but @michaeldhess or @dpb-bah can correct me if I'm wrong on that.

Reply:
Hmmm, it's difficult to say. It does sound out of scope, but I can't think of any other ticket that we've had since I got here that made this many changes to the config files. Ideally things get removed as they become irrelevant, but seeing as these changes were made months ago I'd say a good Spring cleaning is in order.

Reply:
I don't think this was ever resolved.